projects/sample/ReadMe_SampleProject.md

```diff
@@ -2,16 +2,21 @@
 
 The purpose of this sample project is to familiarize you with the tools that are available to you through our **Realtime fMRI Cloud Framework**. In these scripts, we will walk you through a rudimetary project example where we will:
 
-- Initialize a real-time session
+- Simulate starting a projectInterface session on the cloud.
+    - In this example, you will do this from your local computer but under the real experiment circumstances you would start the projectInterface from the cloud.
+- Simulate starting a fileServer session on the console computer.
+    - Again, in this example, you will do this from another terminal window on your local computer but under real experiment circumstances you would start the fileServer from the console computer to initialize a watch for incoming files of interest (e.g., dicoms). 
+- Review the settings on the configuration file that are on the cloud using the web browser.
+- Initialize a real-time experiment session from the web browser.
 - Run a real-time experiment session:
     - that simulates receiving dicoms<sup>\*</sup> in real-time
     - that converts the dicoms received into nifti files
     - but that *only* does a very simple analysis (obtaining the average activation value across all voxels)
     - that plots the values of interest on the internet browser
-- Finalize the real-time session
+- Finalize the real-time experiment session.
 
 ## What's inside the sample project?
-There are six main components to the sample project, which are delineated below. The python scripts are used by the webServer interface to run the project from the web browser, instead of running the project directly from the command line. However, please note that there is an option to run the python scripts from the command line and be able to retrieve remote files afterwards (see [Testing the Sample Project, alternate step 3](https://github.com/paulapbrooks/rt-cloud/tree/sample_dev#testing-the-sample-project)):
+There are six main components to the sample project, which are delineated below. The python scripts are used by the web interface to run the project from the web browser, instead of running the project directly from the command line. However, please note that there is an option to run the python scripts from the command line and be able to retrieve remote files afterwards (see [Testing the Sample Project, alternate step 3](https://github.com/paulapbrooks/rt-cloud/tree/sample_dev#testing-the-sample-project)):
```

Comment:
'The python scripts are used by the web interface to run the project from the web browser' -- This made me think that the web browser was running the python, rather than the web browser start button starts the python running on the cloud computer.
Maybe:
"The python scripts can be started on the cloud computer by using the web interface, or they can be started directly from the command line. ..."